### DISCUSSION TITLE
Private keys are stored unencrypted

### DISCUSSION THREAD

**Discussion by schomatis** [filecoin-project/lotus#8568] [Ideas]
Private keys in the Wallet Keystore are stored in plain text. There are some potential solutions being studied at the moment.

(Incidentally, also the JSON web token is stored in plain text. Whatever solution is applied to the wallet might also be of use for this case.)

**Comment by TippyFlitsUK** [comment]
Hi @schomatis 

Thanks for the report. The issue you are experiencing lies elsewhere.

Moving this issue to Lotus Discussions for help and troubleshooting.

Closing Lotus issue ticket.

**Comment by schomatis** [comment]
> The issue you are experiencing lies elsewhere.

I'm not sure what the above means.

This was a security issue that originated from one of our many audits, and it was acknowledged by pretty much all maintainers back in the day. Doesn't seem like a good candidate for a community discussion.
